tests/queries/test_query.py

```diff
@@ -91,6 +93,35 @@ def test_negated_nullable(self):
         self.assertIsInstance(lookup, IsNull)
         self.assertEqual(lookup.lhs.target, Item._meta.get_field('modified'))
 
+    def test_negated_annotation_nullable(self):
+        query = Query(Item.objects.annotate(year=ExtractYear('modified')).exclude(modified='2021'))
```

Comment:
Greetings, and thanks for your patch. The format of 2021 here is causing a test failure. To make this easier to review could you correct it? Thanks.

Reply:
@jacobtylerwalls I added other test cases. if another test cases are needed, tell me.

Reply:
@jacobtylerwalls could you retry this test 'Tests / JavaScript tests (pull_request) '? I don't know why it was failed.